docs/release-notes/2.28.0.md

```diff
@@ -0,0 +1,112 @@
+
+    As a mature and generally useful feature, this option is now enabled by default, for  easier access to automatic checks and better insight into database health and performance.
+
+    !!! note alert alert-primary "Note"
+        Upgrading to PMM will automatically enable this feature for existing PMM instances. You can disable it at any time from your PMM Advanced Settings.
+
+- **Enhanced Advisor checks**
+
+    PMM 2.28 includes a new major version of Advisors that features some important enhancements. The most significant changes are:
+
+    * Support for multiple queries
+    * Support for Victoria Metrics as a data source
+
+    In a nutshell, these are changes will allow experts to create more intelligent advisor checks so that we can continue delivering more value to your connected PMM instances.
+
+    The file format in which Advisors checks are written has been updated in order to support the new functionality provided by the Advisors service in Percona Platform. 
+
+    This is a breaking change so it is highly recommended to upgrade your PMM to benefit from these enhancements. For more information see [Develop Advisors](https://docs.percona.com/percona-monitoring-and-management/details/develop-checks.html). 
+
+### Ubuntu 22.04 LTS support
+
+
+### Components upgrade
+  - **VictoriaMetrics**: VictoriaMetrics has been upgraded to [1.76.1](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.76.1).
+  - **Node exporter**: Node Exporter has now been updated to `1.1.X`.
+
+
+## New Features
+
+- [PMM-9749](https://jira.percona.com/browse/PMM-9849): Advisors: Possibility to run individual advisor checks separately
+
+- [PMM-9469](https://jira.percona.com/browse/PMM-9469): Advisors: Ability to have multiple queries in single check
+
+- [PMM-9468](https://jira.percona.com/browse/PMM-9468): Advisors: Ability to query VictoriaMetrics as a data source.
+
+
+## Improvements
+
+- [PMM-9839](https://jira.percona.com/browse/PMM-9839): Advisors: Fixed severity naming inconsistency for Critical checks.
+ 
+- [PMM-9841](https://jira.percona.com/browse/PMM-9841): Advisors: Advisor checks are now enabled by default.
+
+- [PMM-8326](https://jira.percona.com/browse/PMM-8326): Advisors: Changed the icon for the Edit Check Rule option with a more suggestive one, that better reflects this functionality.
+
+- [PMM-9362](https://jira.percona.com/browse/PMM-9362): Disconnect PMM and Portal: non-portal user can not perform disconnect
+
+- [PMM-9907](https://jira.percona.com/browse/PMM-9907): pmm2-client now supports Ubuntu 22.04 LTS.
+
+- [PMM-9780](https://jira.percona.com/browse/PMM-9780): VictoriaMetrics has been upgraded to 1.76.1. 
+
+- [PMM-5871](https://jira.percona.com/browse/PMM-5871): Node Exporter has now been updated to 1.1.x.
```

Comment:
we probably don't need this.

Reply:
I mean Node Exported is not needed in RN

Reply:
@JiriCtvrtka @denisok  Please confirm what exactly needs to be mentioned as I am not sure what exactly has changed due to this update.

Reply:
```suggestion
- [PMM-5871](https://jira.percona.com/browse/PMM-5871): Node Exporter has now been updated to 1.3.1.
```